assets/js/app.js

```diff
@@ -113,6 +121,17 @@ window.addEventListener("contextmenu", (event) => {
   }
 });
 
+window.addEventListener("bulk-actions-state", () => {
```

Comment:
This should be more specific, like `lb:session_list:on_selection_change`.

In this case this makes for a neat alternative to hooks, if we start having more of those, we should organise this into another file, but keeping everything here is fine for now :D